modules/primer-utilities/lib/layout.scss

```diff
@@ -81,14 +83,15 @@
 .min-width-0 { min-width: 0 !important; }
 
 // Direction utilities
-
+/* Set the direction to rtl */
 .direction-rtl { direction: rtl !important; }
 
+/* Set the direction to ltr */
```

Comment:
Tiny nitpick, can you remove the whitespace line above to be consistent with other families of utilities?